### PR TITLE
crawler cleanup and lbc ipynb for dummies 

### PR DESCRIPTION
**Pull request by borisborelly**
- paves the way for more crawlers
- starts ipynb for the lbc calendar example
- removed some generated src i was experimenting with

### CODE REVIEW

File: src/michar/cli/app/cmd/crawler.py [hunk 1]
```diff
@@ -27,60 +14,12 @@ def crawl(url: str):
...
-
-
-
-
-
-
-
-    # soup = BeautifulSoup(page.content, "html.parser")
-    # log.debug(soup)
+    krawler: util.Krawl = util.get_crawler(url=url)
```
Comment (borisborelly) [inline src/michar/cli/app/cmd/crawler.py:17]: moved the heavy lifting to `util` to keep this dirt simple

File: src/michar/cli/app/michar.py [hunk 2]
```diff
@@ -20,22 +20,16 @@ def __init__(self, *args, **kwargs):
 @debug_option
 @click.version_option(__version__, prog_name=__name__)
 @click.pass_context
-def gooza(ctx, debug) -> None:
+def gooza(ctx: click.Context, debug) -> None:
     if debug:
         for logger in [logging.getLogger(name) for name in logging.root.manager.loggerDict]:
             logger.setLevel(logging.DEBUG)
-    log.debug(f"Starting micha library ... {__version__}")
+    log.debug(f"Starting michar library ... {__version__}")
```
Comment (borisborelly) [inline src/michar/cli/app/michar.py:27]: FIX

**Review by borisborelly — COMMENTED** [review]
checkout branch in vscode to see the ipynb

File: lbc_calendar.ipynb
Comment (rpuncel) [inline lbc_calendar.ipynb:1]: Gonna assume I don't need to review this

File: tests/michar/test_import.py [hunk 3]
```diff
@@ -0,0 +1,8 @@
+import unittest
```
Comment (rpuncel) [inline tests/michar/test_import.py:1]: unittest? smh

File: src/michar/api/util.py [hunk 4]
```diff
@@ -3,9 +3,21 @@
 from typing import Optional
 from rich.logging import RichHandler
 from rich.traceback import install
-from rich.console import Console
-from rich.theme import Theme
-console = Console(theme=Theme({"logging.level.info": "cyan"}))
+from dataclasses import dataclass, field
+from selenium import webdriver
+from selenium.webdriver.remote import webdriver as webdrv
+from selenium import webdriver
```
Comment (rpuncel) [inline src/michar/api/util.py]: This import happens twice, remove this line

File: src/michar/api/util.py [hunk 5]
```diff
@@ -3,9 +3,21 @@
...
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.webdriver.remote.webdriver import WebDriver as RemoteWebDriver
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support.ui import WebDriverWait
+
+# from selenium.webdriver.chrome.service import Service as BraveService
```
Comment (rpuncel) [inline src/michar/api/util.py]: Delete these commented out lines?

File: src/michar/api/util.py [hunk 6]
```diff
@@ -17,4 +29,70 @@
 
 
 def get_logger(name: Optional[str] = "util") -> logging.Logger:
```
Comment (rpuncel) [inline src/michar/api/util.py:25]: I wanted to talk to you about using structlog, but you beat me to a PR

File: src/michar/cli/app/cmd/crawler.py
Comment (rpuncel) [inline src/michar/cli/app/cmd/crawler.py:1]: It's not doing much yet right?

**Review by rpuncel — COMMENTED** [review]
Looks good, just some questions and food for thought. no need to wait for me

File: src/michar/cli/app/cmd/crawler.py
Comment (borisborelly) [reply src/michar/cli/app/cmd/crawler.py:1]: right, need to actually implement something useful against lbc calendar next

**Pull request merged**